notebooks/tabular_examples/model_agnostic/Simple Kernel SHAP.ipynb

```diff
@@ -69,8 +69,9 @@
     "        ws[len(s)] = ws.get(len(s), 0) + shapley_kernel(M,len(s))\n",
     "        weights[i] = shapley_kernel(M,len(s))\n",
     "    y = f(V)\n",
-    "    tmp = np.linalg.inv(np.dot(np.dot(X.T, np.diag(weights)), X))\n",
-    "    return np.dot(tmp, np.dot(np.dot(X.T, np.diag(weights)), y))\n",
+    "    wsq = np.sqrt(weights)\n",
+    "    result = np.linalg.lstsq(wsq[:, None] * X, wsq * y)\n",
+    "    return result.x\n",
```

Comment:
Shouldn't this be `result[0]` ? Seems like the result is a normal tuple, not a namedTuple or anything with an attribute access.

Please also ensure that you "Restart notebook and run all cells" so the output cells are also refreshed.